scripts/publish

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
I'm assuming this is a straight copy of the publish script from other repos, which is fine.

Reply:
Yep